lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -293,16 +285,12 @@ class CategoryRenderer {
     const headerEl = this.dom.find('.lh-audit-group__header', clumpElement);
     const title = this._clumpTitles[clumpId];
     headerEl.textContent = title;
-    if (description) {
-      const markdownDescriptionEl = this.dom.convertMarkdownLinkSnippets(description);
-      const auditGroupDescription = this.dom.createElement('div', 'lh-audit-group__description');
-      auditGroupDescription.appendChild(markdownDescriptionEl);
-      clumpElement.appendChild(auditGroupDescription);
-    }
+    // if (description) {
```

Comment:
remove?

Reply:
idk, do we want "Run these additional validators on your site to check additional SEO best practices"??

Reply:
I think that is extraneous.  I vote remove this entirely.

Reply:
Keeping for now, it may or may not survive design review.